repograph/repograph.py

```diff
@@ -50,7 +53,16 @@ def iter_submodules(gh, repo):
             click.echo("Can't find submodule %r" % submodule_name, err=True)
             continue
 
-        yield repo_name, submodule_name
+        sub_version = repo.get_file_contents(path).sha
+
+        sub_head = sub_repo.get_commits()[0].sha
+
+        if sub_head != sub_version:
+            comp = sub_repo.compare(sub_version, sub_head)
+        else:
+            comp = None
```

Comment:
It is does not take a long time to do the compare, also there is `status` field, which is `identical` when the submodule pointer is up-to-date.

Will change the code to always make the `compare` call.